### PR TITLE
Improvement: Show playlist item count in header

### PR DESCRIPTION
**Pull request by wutschel**
## Description

Closes https://github.com/xbmc/Official-Kodi-Remote-iOS/issues/976.

Adds the amount of items for a playlist to the playlist header, e.g. "Playlist (16)".

Screenshots: https://ibb.co/2PT6G4Q

## Summary for release notes

Improvement: Show playlist item count in header

### CODE REVIEW

File: XBMC Remote/NowPlaying.m [hunk 1]
```diff
@@ -121,6 +121,15 @@ - (UIImage*)resizeToolbarThumb:(UIImage*)img {
 
 #pragma mark - utility
 
+- (NSString*)getPlaylistHeaderLabel {
+    NSString *headerLabel = LOCALIZED_STR(@"Playlist");
+    int numItems = (int)playlistData.count;
+    if (numItems > 0) {
+        headerLabel = [NSString stringWithFormat:@"%@ (%d)", headerLabel, numItems];
```
Comment (kambala-decapitator) [inline XBMC Remote/NowPlaying.m:128]:
```suggestion
    NSUInteger numItems = playlistData.count;
    if (numItems > 0) {
        headerLabel = [NSString stringWithFormat:@"%@ (%lu)", headerLabel, numItems];
```

**Review by kambala-decapitator — APPROVED** [review]

File: XBMC Remote/NowPlaying.m (hunk 1 above)
Comment (wutschel) [reply XBMC Remote/NowPlaying.m]: Agreed

File: XBMC Remote/NowPlaying.m [hunk 2]
```diff
@@ -121,6 +121,15 @@ - (UIImage*)resizeToolbarThumb:(UIImage*)img {
 
 #pragma mark - utility
 
+- (NSString*)getPlaylistHeaderLabel {
+    NSString *headerLabel = LOCALIZED_STR(@"Playlist");
+    NSUInteger numItems = playlistData.count;
+    if (numItems > 0) {
+        headerLabel = [NSString stringWithFormat:@"%@ (%ld)", headerLabel, numItems];
```
Comment (kambala-decapitator) [inline XBMC Remote/NowPlaying.m:128]: you should start letting GitHub commit suggestions and then just rebase on your side :)

```suggestion
        headerLabel = [NSString stringWithFormat:@"%@ (%lu)", headerLabel, numItems];
```

Comment (wutschel) [reply XBMC Remote/NowPlaying.m:128]: The devil is in the details.

**Pull request merged**